### PR TITLE
Initial version of acceptance check scripts

### PR DESCRIPTION
**Pull request by bcouturi**
Very crude test extending D0 trajectory following either P or the vector between primary vertex and end vertex. Needs to be improved to take the uncertainty on vertices and momentum into account.

### CODE REVIEW

**Comment by particleist** [comment]
These are details but I found some of the notation hard to follow. E.g. here

+Double_t RadialSelector::GetZforRadius(Double_t R,
-                     Double_t px, Double_t py, Double_t pz,
-                     Double_t vx, Double_t vy, Double_t vz) {

px,py,pz would lead the reader to think they are momenta, but these variables actually hold the PV positions. Similarly why is this
-  auto u = p;

actually needed? I think the code would be more readable if the variable names were a bit more verbose, basically.

**Comment by bcouturi** [comment]
You're right. I cleaned up and documented.
Not sure about how to deal with uncertainties with what we have in the ntuple though.

**Comment by particleist** [comment]
Thanks Ben!

I am confused by the maths here. 

+Double_t RadialSelector::GetRadius(Double_t z0,
-                 const TVector3 &pv,
-                 const TVector3 &ev) {
-  Double_t tmp = (z0 - pv.Z()) / ev.Z();
-  Double_t rad = sqrt(pow(pv.X() + tmp \* ev.X(), 2)
-            + pow( pv.Y() + tmp \* ev.Y(), 2));
-  return rad;
  +}

why do you sum up pv.X/Y and ev.X/Y, multiplied by this "how far I travelled" factor? I am also not sure about the calculation of the factor for how far you travelled. Could you explain the reasoning please?

**Comment by bcouturi** [comment]
Lack of ? :-) Went through quickly last night and shouldn't have.
The method just finds out the radius at given z for a line defined by a point and direction.
Cleaning up and pushing. Sorry for that.

**Comment by particleist** [comment]
Thanks Ben, but I don't see the revised commit?

**Comment by bcouturi** [comment]
There you go...

**Comment by particleist** [comment]
Ah sorry, stupid browser cache issue. Merged now!
